core/lib/zksync_core/src/api_server/web3/backend_jsonrpsee/namespaces/zks.rs

```diff
@@ -20,6 +20,7 @@ use zksync_web3_decl::{
 use crate::api_server::web3::{backend_jsonrpsee::into_jsrpc_error, ZksNamespace};
 
 #[async_trait]
+
```

Comment:
Looks like a bogus change to me; why is it necessary?

Reply:
Removed.